### DISCUSSION TITLE
Help with Browsersync

### DISCUSSION THREAD

**Discussion by colma** [tailpress/tailpress#203] [Q&A] [answered]
Hey,

I'm trying to enable Browsersynch with the latest release. I saw an [older thread](https://github.com/jeffreyvr/tailpress/discussions/108) with the same issue, but the package.json has changed since. Any tips on how to use / enable Browsersync with Tailpress?

**Comment by jeffreyvr** [comment, marked as answer]
Hi! If you use the default setup with Laravel Mix, you can uncomment this portion:

https://github.com/jeffreyvr/tailpress/blob/master/webpack.mix.js#L21:L26

**Reply by colma** [reply]
Awesome, that was easier than I thought :) Thank you!
